Backpack/dokka.gradle

```diff
@@ -1,63 +0,0 @@
-/**
- * Backpack for Android - Skyscanner's Design System
- *
- * Copyright 2018-2021 Skyscanner Ltd
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-tasks.register("generateDokkaStatics") {
```

Comment:
We had a few static sites that we generated manually - I'm not sure they really provide a lot of value. For example we had the version number in the path and a redirect for "latest". However we only kept the documentation for the latest version anyways, so this doesn't really provide any value. I removed all the versioning config instead which then doesn't require all the other index pages etc.